test/enigma_test.rb

```diff
@@ -0,0 +1,26 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/enigma"
+
+class EnigmaTest < Minitest::Test
+  def test_enigma_class_exists
+    e = Enigma.new
+
+    assert_instance_of Enigma, e
+  end
+
+  def test_enigma_can_encrypt_one_letter
+    e = Enigma.new
+    ciph = e.encrypt_letter("a", 3)
+
+    assert_equal "d", ciph
+  end
+
+  def test_encrypt_method_takes_arguements
```

Comment:
If you are having trouble thinking about how to test the output of a method, it might be that the method is doing too much.